apps/workflows/jobs/refill-daily.ts

```
@@ -12,30 +12,40 @@ client.defineJob({
 
-  run: async (_payload, io, _ctx) => {
+  run: async (payload, io, _ctx) => {
+    const date = payload.ts;
+    // Set up last day of month so if refillDay is after last day of month, Key will be refilled today.
+    const lastDayOfMonth = new Date(date.getFullYear(), date.getMonth() + 1, 0).getDate();
+    const today = date.getDate();
     const db = connectDatabase();
     const tb = new Tinybird(env().TINYBIRD_TOKEN);
-    const t = new Date();
-    t.setUTCHours(t.getUTCHours() - 24);
+    let sql = ""
```

Biome

[error] This let declares a variable that is only assigned once.

'sql' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)